src/pages/Artist.jsx

```diff
@@ -26,13 +31,21 @@ function Artist() {
   }, [search]);
 
   useEffect(() => {
-    const timeoutId = setTimeout(() => {
-      setIndex((index + 1) % artworks.length);
-    }, 3000);
-    return () => clearTimeout(timeoutId);
-  }, [index, artworks]);
+    if (!isOpen) {
+      const timeoutId = setTimeout(() => {
+        setIndex((index + 1) % artworks.length);
+      }, 3000);
+      return () => clearTimeout(timeoutId);
+    }
+  }, [index, artworks, isOpen]);
 
   const artistName = artworks[0]?.artist_display || '';
+  const artworkTitle = artworks[index]?.title || '';
+  const artworkDisplay = artworks[index]?.date_display || '';
+  const artworkImage = artworks[index]
+    ? `https://www.artic.edu/iiif/2/${artworks[index].image_id}/full/400,/0/default.jpg`
+    : '';
```

Comment:
Tu peux faire pareil qu'au dessus avec le `?.`; ça évite de faire une ternaire

```suggestion
  const artworkImage = `https://www.artic.edu/iiif/2/${artworks[index]?.image_id}/full/400,/0/default.jpg` || '';
```